### DISCUSSION TITLE
How to use code templates signed by 3 or more users?

### DISCUSSION THREAD

**Discussion by pakkers** [projectdiscovery/nuclei-templates#11030] [General]
Let's suppose this situation: 
- Machine A with nuclei that works as a centralised machine for templates
- Machine B, C, etc. with templates signed by user B, C, etc. 
In machine A (.config/nuclei/keys) we have A.pub.crt, B.pub.crt, C.pub.crt, etc.

In this situation (if I'm not wrong), Machine A (with pub keys of B, C, etc.) only can execute templates by project discovery (because they are integrated in the installation) and  it's own templates. 
But if from A we want to run templates of user B, we need to export B public key to use it. If we do this, we cannot execute templates of user A. 

export NUCLEI_USER_CERTIFICATE=~./config/nuclei/keys 
don't work to include A.pub.crt, B.pub.crt, C.pub.crt in nuclei checks for signed templates.

Am I doing something wrong ? 

thanks for all.

**Comment by princechaddha** [comment]
Hi @pakkers, the response time to this thread was longer than usual and it was somehow overlooked. Someone from the dev team will reply on this shortly.
